packages/grafana-ui/src/components/Select/getSelectStyles.ts

```diff
@@ -57,6 +57,12 @@ export const getSelectStyles = stylesFactory((theme: GrafanaTheme2) => {
     optionSelected: css`
       background: ${theme.colors.action.selected};
     `,
+    optionDisabled: css`
+      label: grafana-select-option-disabled;
```

Comment:
TIL about emotion label https://emotion.sh/docs/labels

Do we actually need it?

Reply:
I added it for consistency, but it also helps to identify a component in the DOM when inspecting elements.